[2022-07-26 03:15:26,060] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: GOOG_historic_data.create_historic_table manual__2022-07-26T03:15:24.897719+00:00 [queued]>
[2022-07-26 03:15:26,069] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: GOOG_historic_data.create_historic_table manual__2022-07-26T03:15:24.897719+00:00 [queued]>
[2022-07-26 03:15:26,069] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 03:15:26,069] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-26 03:15:26,069] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 03:15:26,084] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-26 03:15:24.897719+00:00
[2022-07-26 03:15:26,088] {standard_task_runner.py:52} INFO - Started process 33213 to run task
[2022-07-26 03:15:26,090] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'GOOG_historic_data', 'create_historic_table', 'manual__2022-07-26T03:15:24.897719+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmptyq_rt05', '--error-file', '/tmp/tmpb5czzsl5']
[2022-07-26 03:15:26,092] {standard_task_runner.py:80} INFO - Job 162: Subtask create_historic_table
[2022-07-26 03:15:26,177] {task_command.py:371} INFO - Running <TaskInstance: GOOG_historic_data.create_historic_table manual__2022-07-26T03:15:24.897719+00:00 [running]> on host 527e7914f635
[2022-07-26 03:15:26,265] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=GOOG_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-26T03:15:24.897719+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-26T03:15:24.897719+00:00
[2022-07-26 03:15:26,281] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-26 03:15:26,286] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__GOOG (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            PRIMARY KEY (datetime)
        )
    , parameters: None
[2022-07-26 03:15:26,296] {python.py:173} INFO - Done. Returned value was: None
[2022-07-26 03:15:26,318] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=GOOG_historic_data, task_id=create_historic_table, execution_date=20220726T031524, start_date=20220726T031526, end_date=20220726T031526
[2022-07-26 03:15:26,385] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-26 03:15:26,512] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
